frontend/src/pages/ActivityReport/Pages/topicsResources.js

```diff
@@ -54,7 +54,8 @@ const TopicsResources = ({
             <br />
             Enter the URL for OHS resource(s) used.
             {' '}
-            <a target="_blank" rel="noreferrer" href="https://eclkc.ohs.acf.hhs.gov/">https://eclkc.ohs.acf.hhs.gov/</a>
+            {/* eslint-disable-next-line react/jsx-no-target-blank */}
```

Comment:
Even with `rel="noopener"` eslint still complains because of [react/jsx-no-target-blank](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md) hence the ignore/